core/controllers/question_editor_test.py

```diff
@@ -257,7 +257,7 @@ def test_post_with_admin_email_allows_question_linking(self):
             self.assertEqual(
                 question_summaries[0].id, self.question_id)
             self.assertEqual(
-                skill_descriptions[0], 'Skill Description')
+                skill_descriptions[0], ['Skill Description'])
```

Comment:
I feel like there's some misnaming here; I would expect skill_descriptions[0] to be a skill_description which is presumably a string.

Reply:
Done.